package.json

```diff
@@ -6,20 +6,26 @@
   "dependencies": {
     "core-util-is": "~1.0.0",
     "inherits": "~2.0.1",
+    "inline-process-browser": "~2.0.1",
     "isarray": "~1.0.0",
     "process-nextick-args": "~1.0.6",
     "string_decoder": "~0.10.x",
+    "unreachable-branch-transform": "~0.5.0",
```

Comment:
No, they are used to avoid bundling unneeded stuff for the browserify build.
Are you saying that these adds up in the browserify build?